pkg/kubectl/cmd/util/openapi/doc.go

```diff
@@ -18,4 +18,4 @@ limitations under the License.
 // from a Kubernetes server and then indexing the type definitions.
 // The openapi spec contains the object model definitions and extensions metadata
 // such as the patchStrategy and patchMergeKey for creating patches.
-package openapi
+package openapi // k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi
```

Comment:
@apelisse I believe it is a part of https://github.com/kubernetes/kubernetes/issues/68231.

This is necessary so that if someone wants to import only this package (should be rare?), the import path is not referred to as github.com, but as the k8s.io vanity url.
Ref: https://golang.org/doc/go1.4#canonicalimports